subsys/bluetooth/host/classic/ssp.c

```diff
@@ -443,6 +443,24 @@ void bt_hci_link_key_notify(struct net_buf *buf)
 
 	LOG_DBG("%s, link type 0x%02x", bt_addr_str(&evt->bdaddr), evt->key_type);
 
+	if (IS_ENABLED(CONFIG_BT_SMP_SC_ONLY) &&
+		(!(evt->key_type == BT_LK_AUTH_COMBINATION_P256))) {
```

Comment:
Please fix your split line indentation and alignement. It should be N tabs + 0-7 spaces so that you align right after the `(` of the previous line. The clang-format inline annotations point this out as well.

Reply:
Yes. I noticed it. I recently discovered that Zephyr's tab occupies 8 character positions.

Reply:
Updated.